assets/wizards/popups/views/popup-group/index.js

```diff
@@ -59,22 +61,36 @@ const descriptionForPopup = (
 const PopupGroup = ( {
 	deletePopup,
 	items: { active = [], draft = [], test = [], inactive = [] },
+	manageCampaignGroup,
 	previewPopup,
 	setTermsForPopup,
 	setSitewideDefaultPopup,
 	publishPopup,
 	updatePopup,
 	segments,
+	settings,
 } ) => {
+	const activeCampaignGroup = settings.reduce(
+		( acc, { key, value } ) => ( key === 'newspack_popups_active_campaign_group' ? value : acc ),
+		null
+	);
 	const [ campaignGroup, setCampaignGroup ] = useState( -1 );
 	const [ campaignGroups, setCampaignGroups ] = useState( -1 );
+	const [ segmentId, setSegmentId ] = useState();
+	const [ previewPopoverIsVisible, setPreviewPopoverIsVisible ] = useState();
 
 	useEffect( () => {
 		apiFetch( {
 			path: '/wp/v2/newspack_popups_taxonomy?_fields=id,name',
 		} ).then( terms => setCampaignGroups( terms ) );
 	}, [] );
 
+	useEffect( () => {
+		if ( -1 === campaignGroup && activeCampaignGroup > 0 ) {
+			setCampaignGroup( activeCampaignGroup );
```

Comment:
I think we need to cast this term ID as a number here to ensure the `filteredByGroup` function always works:

```suggestion
			setCampaignGroup( parseInt( activeCampaignGroup ) );
```

I sometimes run into an issue where if I refresh the page or navigate to the Campaigns tab after setting an active group, the initial page load would show an empty list even if there were campaigns in the active group.

Reply:
I wasn't able to reproduce this on my local instance, but I _think_ I saw it on a production site I deployed this branch to. Here's the fix: https://github.com/Automattic/newspack-plugin/pull/775/commits/7b91839c9d8397081e1441c1996107bba6d22bf0

Reply:
Thanks!